### ISSUE TITLE
Add context to the footprint results

### ISSUE THREAD

**Issue by KKulma** [KKulma/footprint-shiny-app#4]
the calculated footprint should be explained in more tangible terms, e.g. allowable carbon budget, car miles, melted ice. Add more examples and resources. Some examples

- [m2 of melted Arctic ice](https://shameplane.com/?fromCity=Stockholm&fromCode=ARN&toCity=Bangkok&toCode=DMK&roundtrip=true&typeofseat=3) 

- [other means of transport](https://flightfree.org/flight-emissions-calculator)
- [various equivalents](https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator) with [underlying data](https://www.epa.gov/energy/greenhouse-gases-equivalencies-calculator-calculations-and-references)

**Comment by acircleda** [comment]
According to [this paper](https://science.sciencemag.org/content/354/6313/747.full):

> First, the observed linear relationship allows us to estimate a sensitivity of 3.0 ± 0.3 m2 of September Arctic sea-ice loss per metric ton of anthropogenic CO2 emissions during the observational period 1953 to 2015.

So, we can divide emissions output (in kg) by 1000 to get metric tons, and then multiple by 3.0 to get m2. We can divide this by 2.59 to get square miles (in parentheses) for people who don't use the metric system

**Comment by acircleda** [comment]
According to the [EPA](https://www.epa.gov/energy/greenhouse-gases-equivalencies-calculator-calculations-and-references):

> A medium growth coniferous or deciduous tree, planted in an urban setting and allowed to grow for 10 years, sequesters 23.2 and 38.0 lbs of carbon, respectively.

Their calculation *0.060 metric ton CO2 per urban tree planted*

We would have to use the output in CO2. It does not say on the EPA website whether this includes RF, so we may need to look more carefully at this: https://www3.epa.gov/climatechange/Downloads/method-calculating-carbon-sequestration-trees-urban-and-suburban-settings.pdf

**Comment by KKulma** [comment]
> According to [this paper](https://science.sciencemag.org/content/354/6313/747.full):
> 
> > First, the observed linear relationship allows us to estimate a sensitivity of 3.0 ± 0.3 m2 of September Arctic sea-ice loss per metric ton of anthropogenic CO2 emissions during the observational period 1953 to 2015.
> 
> So, we can divide emissions output (in kg) by 1000 to get metric tons, and then multiple by 3.0 to get m2. We can divide this by 2.59 to get square miles (in parentheses) for people who don't use the metric system

Really like this idea 👍

**Comment by KKulma** [comment]
> According to the [EPA](https://www.epa.gov/energy/greenhouse-gases-equivalencies-calculator-calculations-and-references):
> 
> > A medium growth coniferous or deciduous tree, planted in an urban setting and allowed to grow for 10 years, sequesters 23.2 and 38.0 lbs of carbon, respectively.
> 
> Their calculation _0.060 metric ton CO2 per urban tree planted_
> 
> We would have to use the output in CO2. It does not say on the EPA website whether this includes RF, so we may need to look more carefully at this: https://www3.epa.gov/climatechange/Downloads/method-calculating-carbon-sequestration-trees-urban-and-suburban-settings.pdf

Absolutely, but to start with we could focus on "low-hanging fruit" - examples that are more straightforward to use in our case?

**Comment by acircleda** [comment]
> > According to the [EPA](https://www.epa.gov/energy/greenhouse-gases-equivalencies-calculator-calculations-and-references):
> > > A medium growth coniferous or deciduous tree, planted in an urban setting and allowed to grow for 10 years, sequesters 23.2 and 38.0 lbs of carbon, respectively.
> > 
> > 
> > Their calculation _0.060 metric ton CO2 per urban tree planted_
> > We would have to use the output in CO2. It does not say on the EPA website whether this includes RF, so we may need to look more carefully at this: https://www3.epa.gov/climatechange/Downloads/method-calculating-carbon-sequestration-trees-urban-and-suburban-settings.pdf
> 
> Absolutely, but to start with we could focus on "low-hanging fruit" - examples that are more straightforward to use in our case?

We can simply use this as an estimate and say "it would take xxx newly planted trees to make up for your flight".

**Comment by acircleda** [comment]
We can also show the proportion a flight takes up of one's personal carbon footprint, based on [this](https://www.iges.or.jp/en/pub/15-degrees-lifestyles-2019/en):

>In terms of all GHGs, the ranges of the estimated lifestyle carbon footprint targets for 2030, 2040, and 2050 are respectively 3.2–2.5, 2.2–1.4, and 1.5–0.7 tCO2e per capita.

We can display this in terms of the 2030 target and show where it calls in the 3.2-2.5 range.

**Comment by acircleda** [comment]
This is a first pass, and its not very pretty, but it could be something like this:
![image](https://user-images.githubusercontent.com/51770624/104245569-7d2aff80-5432-11eb-8272-d297000e3eee.png)

**Comment by acircleda** [comment]
This is not the prettiest, and its a little hacky, but something like this? It may also take up too much space:

![image](https://user-images.githubusercontent.com/51770624/104259418-6646d600-544f-11eb-96ab-54b63beaed16.png)

**Comment by acircleda** [comment]
![image](https://user-images.githubusercontent.com/51770624/104260408-72cc2e00-5451-11eb-8ab7-fcf1033d3459.png)

**Comment by acircleda** [comment]
I pushed these to the visuals branch
